src/main/java/com/teamrocket/tms/models/entities/Project.java

```diff
@@ -27,7 +27,7 @@ public class Project {
     @Column(name = "description", length = 250)
     private String description;
 
-    @OneToOne
+    @Transient
     private Team team;
```

Comment:
De ce e @Transient? Nu as vrea sa vad in DB asta? 
Si pare ca nu e facuta si invers, de la Team la project.